src/standalone/Soil/Soil.jl

```diff
@@ -88,7 +88,8 @@ import ClimaLSM:
     surface_emissivity,
     surface_air_density,
     surface_height,
-    surface_resistance
+    surface_resistance,
+    get_drivers
```

Comment:
do we need to extend `get_drivers` for the Soil model?

Reply:
I think since each model can store the drivers in a different spot (e.g. bucket.atmos, bucket.radiation, vs the EnergyHydrology model storing them in the boundary condition) we do need a different method for most models. but e.g. RichardsEquation does not need a new method since it never uses the drivers.